content/examples/databricks-cluster-local.Rmd

```diff
@@ -0,0 +1,170 @@
+---
+title: "Option 2 - Working inside of Databricks"
+output:
+  html_document:
+    toc: true
+    toc_depth: 2
+    toc_float:
+      collapsed: false
+params:
+  width: 600
+---
+
+## Overview
+
+If the recommended path of connecting to Spark remotely with Databricks Connect
+does not apply to your use case, then you can install RStudio Server Pro
+directly within a Databricks cluster.
+
+With this configuration, RStudio Server Pro is installed on the Spark driver
+node and allows users to work locally with Spark.
+
+<img src="/images/deployment/databricks/rstudio-databricks-local.png" width='800px' align='center'/>
+
+This configuration can result in increased complexity, limited connectivity to
+other storage and compute sources, resource contention between RStudio Server
+Pro and Databricks, and maintenance concerns due to the ephemeral nature of
+Databricks clusters.
+
+For additional details, refer to the
+[FAQ for RStudio in the Databricks Documentation](https://docs.databricks.com/spark/latest/sparkr/rstudio.html#frequently-asked-questions-faq).
+
+## Advantages and limitations
+
+Advantages:
+
+- Ability for users to connect `sparklyr` to Spark without configuring remote
+  connectivity
+- Can load data from the cluster directly into an R session since RStudio Server
+  Pro is installed within the Databricks cluster
+
```

Comment:
```suggestion
- High bandwidth connection between the R and Spark's JVM processes because they are running on the same machine.
```

Reply:
Added this to the list in c9f784b, thanks!